ports/tailspin/port.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 3,
+  "name": "tailspin.zip",
+  "items": [
+    "tailspin.sh",
+    "tailspin"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Tailspin!",
+    "porter": [
+      "Slobters"
+    ],
+    "desc": "Play as a treasure seeking canary and drill through 4 unique worlds of action packed platforming. Can you survive the infested depths below and attain everlasting glory?",
+    "desc_md": null,
+    "inst": "Ready to run!",
+    "inst_md": null,
+    "genres": [
+      "platformer"
+    ],
+    "image": null,
+    "rtr": true,
+    "exp": false,
+    "runtime": null,
+    "reqs": [
+      "power"
+    ],
+    "arch": [
+      "armhf"
```

Comment:
If I'm not mistaken, I believe gmloadernext is compiled for aarch64.

Reply:
Fixed.